src/symbol/symbol_size.js

```diff
@@ -31,17 +31,17 @@ export type SizeData = {
 // the painter to set symbol-size-related uniforms
 function getSizeData(tileZoom: number, layer: SymbolStyleLayer, sizeProperty: string): SizeData {
     const declaration: StyleDeclaration = layer.getLayoutDeclaration(sizeProperty);
-    const isFeatureConstant = !declaration || declaration.expression.isFeatureConstant;
+    const isFeatureConstant = !declaration || declaration.isFeatureConstant();
 
-    if (!declaration || declaration.expression.isZoomConstant) {
+    if (!declaration || declaration.isZoomConstant()) {
         return isFeatureConstant ? {
             functionType: 'constant',
             layoutSize: layer.getLayoutValue(sizeProperty, {zoom: tileZoom + 1})
         } : { functionType: 'source' };
     }
 
     // calculate covering zoom stops for zoom-dependent values
-    const levels = declaration.expression.zoomStops;
+    const levels = (declaration.expression: any).zoomStops;
```

Comment:
This `any` cast is justified by the `!declaration || declaration.isZoomConstant()` check above, right?  Let's include a line comment indicating that.